src/System.Runtime.InteropServices.PInvoke/ref/System.Runtime.InteropServices.PInvoke.cs

```diff
@@ -0,0 +1,403 @@
+        [System.Security.SecurityCriticalAttribute]
+        public static System.IntPtr GetFunctionPointerForDelegate(System.Delegate d) { return default(System.IntPtr); }
+        [System.Security.SecurityCriticalAttribute]
+        public static System.IntPtr GetFunctionPointerForDelegate<TDelegate>(TDelegate d) { return default(System.IntPtr); }
+        [System.Security.SecurityCriticalAttribute]
+        public static int GetLastWin32Error() { return default(int); }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        [System.ObsoleteAttribute("OffsetOf(Type, string) may be unavailable in future releases. Instead, use OffsetOf<T>(string). For more info, go to http://go.microsoft.com/fwlink/?LinkID=296511")]
+        public static System.IntPtr OffsetOf(System.Type t, string fieldName) { return default(System.IntPtr); }
+        public static System.IntPtr OffsetOf<T>(string fieldName) { return default(System.IntPtr); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringAnsi(System.IntPtr ptr) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringAnsi(System.IntPtr ptr, int len) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringUni(System.IntPtr ptr) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringUni(System.IntPtr ptr, int len) { return default(string); }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        [System.ObsoleteAttribute("PtrToStructure(IntPtr, Object) may be unavailable in future releases. Instead, use PtrToStructure<T>(IntPtr). For more info, go to http://go.microsoft.com/fwlink/?LinkID=296512")]
+        [System.Security.SecurityCriticalAttribute]
+        public static void PtrToStructure(System.IntPtr ptr, object structure) { }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        [System.ObsoleteAttribute("PtrToStructure(IntPtr, Type) may be unavailable in future releases. Instead, use PtrToStructure<T>(IntPtr). For more info, go to http://go.microsoft.com/fwlink/?LinkID=296513")]
+        [System.Security.SecurityCriticalAttribute]
+        public static object PtrToStructure(System.IntPtr ptr, System.Type structureType) { return default(object); }
+        [System.Security.SecurityCriticalAttribute]
+        public static T PtrToStructure<T>(System.IntPtr ptr) { return default(T); }
+        [System.Security.SecurityCriticalAttribute]
+        public static void PtrToStructure<T>(System.IntPtr ptr, T structure) { }
```

Comment:
The naming makes sense to me. We use `<structureType>` when the structure parameter is of type System.Type, and `<structure>` when it is of type System.Object or generic <T>.

As for `<t>`, I can rename it to `<type>` perhaps.